crates/store/re_protos/proto/rerun/v1alpha1/manifest_registry.proto

```diff
@@ -131,7 +131,7 @@ message RegisterWithDatasetRequest {
 }
 
 message RegisterWithDatasetResponse {
-  rerun.common.v1alpha1.TaskId id = 1;
+  rerun.common.v1alpha1.DataframePart data = 1;
```

Comment:
Please specify the expected schema for this dataframe as an extension, similar to what we do for `CreatePartitionsManifestResponse`:
https://github.com/rerun-io/rerun/blob/1bdf400d8e0f53df1e851d2557880234b1c6cd49/crates/store/re_protos/src/v1alpha1/rerun.manifest_registry.v1alpha1.ext.rs#L283-L352